Helper/Elasticsearch/ClientRequest.php

```diff
@@ -257,13 +257,13 @@ public function getHierarchy(string $emsKey, string $childrenField, int $depth =
             return null;
         }
 
-        $out = new HierarchicalStructure($item['_type'], $item['_id'], $item['_source']);
+        $out = new HierarchicalStructure($item['_type'], $item['_id'], $item['_source'], ($item['_id'] === $activeChild ? true : false));
```

Comment:
Pass the activeChild as parameter, and do the logic and "setActive" in the constructor